src/components/Sidebar/Sidebar.js

```diff
@@ -1,25 +1,29 @@
 // @flow strict
 import React from 'react';
-import Contacts from './Contacts';
-import Copyright from './Copyright';
 import Menu from './Menu';
 import Logo from './Logo';
 import styles from './Sidebar.module.scss';
 import { useSiteMetadata } from '../../hooks';
 
 const Sidebar = () => {
   const {
-    author, logo, copyright, menu
+    logo, menu
   } = useSiteMetadata();
 
   return (
     <div className={styles['sidebar']}>
-      <div className={styles['sidebar__inner']}>
-        <Logo img={logo} />
-        <Menu menu={menu} />
-        <Contacts contacts={author.contacts} />
-        <Copyright copyright={copyright} />
-      </div>
+      <nav className={styles['sidebar__inner']}>
+        <ul className={styles['sidebar__inner_list']}>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Logo img={logo} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+            <Menu menu={menu} />
+          </li>
+          <li className={styles['sidebar__inner_list_item']}>
+          </li>
```

Comment:
Why did you keep the empty list element?

Reply:
easiest way to center the menu

Reply:
I prefer the menu on the right hand side, just like on notion.so